src/ontology/uberon-edit.obo

```diff
@@ -224055,6 +224055,14 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 relationship: part_of UBERON:0006767 ! head of femur
 property_value: dcterms-date "2023-05-30T06:34:18Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480063
+name: dorsal iris
```

Comment:
If "dorsal rim of the iris" is an exact synonym, consider adding (dbxref [PMID:29579253](https://pubmed.ncbi.nlm.nih.gov/29579253)).